packages/hash/api/src/model/entity.model.ts

```diff
@@ -414,10 +414,21 @@ class __Entity {
     return outgoingDbLinks.map((dbLink) => new Link(dbLink));
   }
 
-  async getIncomingLinks(client: DbClient) {
+  async getIncomingLinks(
+    client: DbClient,
+    params?: {
+      activeAt?: Date;
+      stringifiedPath?: string;
+      path?: PathComponent[];
+    },
+  ) {
+    const { activeAt, stringifiedPath, path } = params || {};
+
     const incomingDbLinks = await client.getEntityIncomingLinks({
       accountId: this.accountId,
       entityId: this.entityId,
+      activeAt,
+      path: stringifiedPath ?? (path ? Link.stringifyPath(path) : undefined),
```

Comment:
My bad, this was something I needed before the graph/api to get a comment's replies. When I looked at it I thought it was `knowledge/entity.model.ts` instead of the old one. Reverted in 07b1ccb7997ffa9868b81442af3f4dd3c3f5dbdb